src/Sfa.Tl.ResultsAndCertification.Web/Mapper/AdminChangeLogMapper.cs

```diff
@@ -119,7 +119,7 @@ private static string FormatDateTime(DateTime dateTime)
             => $"{dateTime:d MMMM yyyy h:mm}{dateTime.ToString("tt").ToLower()}";
 
         private static string FormatDateTime2(DateTime dateTime)
-            => $"{dateTime:d MMMM yyyy, hh:mm}{dateTime.ToString("tt").ToLower()}";
+            => $"{dateTime:d MMMM yyyy, h:mm}{dateTime.ToString("tt").ToLower()}";
```

Comment:
Why are we keeping two methods to format dates when only one is used now to display formatted dates? The old one is only used to create the link text. Why not use the change log id instead?

Reply:
There is a difference between the way the link is displayed and the one on the page header.